cmake/gfortran_flags.cmake

```diff
@@ -0,0 +1,88 @@
+# ================================
+# Set gfortran compile flags
+# ================================
+message(STATUS "Getting gfortran flags")
+
+# Set flags for all build types
+set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -std=f2008ts -cpp -ffree-line-length-none -fall-intrinsics")
```

Comment:
`-fstd=f2008ts` is a [deprecated flag](https://gcc.gnu.org/onlinedocs/gfortran/Fortran-Dialect-Options.html). In general I've found there are bugs in Gfortran that makes these standard flags problematic (e.g. Gfortran 9.3.0 has -fstd bugs that make lots of spurious warnings). 

Is it better to not use -fstd except if the builder specifies them on the CMake command line?